lib/config/definitions.js

```diff
@@ -159,6 +159,20 @@ const options = [
     type: 'string',
     admin: true,
   },
+  {
+    name: 'user',
```

Comment:
I think let's go with verbose "username" and "password" options

Reply:
can i have short variants of verbose cli args?

Reply:
fixed